trigger.py

```
@@ -0,0 +1,235 @@
+        # 1. Note value (0-127)
+        midi_data.append(note)
+        
+        # 2. Velocity value (0-127)
+        midi_data.append(velocity)
+        
+        # 3. Length whole part (0-127)
+        length_whole = min(127, int(length))
+        midi_data.append(length_whole)
+        
+        # 4. Length decimal part (0-9)
+        length_decimal = int(round((length - length_whole) * 10)) % 10
```

Ruff (0.12.2)

Value being cast to `int` is already an integer

Remove unnecessary `int` call

(RUF046)